index.html

```diff
@@ -24,6 +24,7 @@
     <!-- Add Roboto font and Google icons -->
     <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Roboto:300,400,500,700,400italic">
     <link rel="stylesheet" href="//fonts.googleapis.com/icon?family=Material+Icons">
+    <link href="https://fonts.googleapis.com/css?family=Fjalla+One" rel="stylesheet">
```

Comment:
Are we using this everywhere? If so we can remove Roboto?

Reply:
this is just the font for the "agriboard" title :) roboto is the font for everything else